lib/funiq/Settings.h

```diff
@@ -8,13 +8,15 @@ struct Settings
 		caseInsensitive(false),
 		showAllMatches(false),
 		showTotals(false),
-		ignoreNonAlphaNumeric(false){}
+		ignoreNonAlphaNumeric(false),
+		totalsFieldWidth(7){} // count-field width 7 used by GNU uniq
```

Comment:
I'd probably want to avoid having this value represented in the settings if we don't intend for it to be configurable (and I don't see the benefit in allowing a user to configure this for the time being - uniq does not). If we decided to make it configurable at a later date for whatever reason, we could put it in the settings then.

Until that happens, I think I'd be happier with a `TOTALS_FIELD_WIDTH` const inside `Matcher.h`. What do you think?

Reply:
Sounds good, I'll const it. I only put it in there as a "just in case", but given that even uniq doesn't make that configurable, I don't think there's ever been a pressing use case for it irl.